app/models/ownership.rb

```diff
@@ -4,16 +4,7 @@ class Ownership < ActiveRecord::Base
 
   validates :user_id, uniqueness: { scope: :rubygem_id }
 
-  before_destroy :keep_last_owner
-
-  private
-
-  def keep_last_owner
-    if rubygem.owners.count == 1
-      errors[:base] << "Can't delete last owner of a gem."
-      false
-    else
-      true
-    end
+  def safe_destroy
+    rubygem.owners.count != 1 && destroy ? true : false
```

Comment:
Ok, that makes sense. What about to rename this to something better describing the usage?

You can move

```ruby
rubygem.owners.count != 1
```
somewhere else to just call something like

```ruby
rubygem.many_owners? && destroy
```

Also is there chance that gem has 0 owners?

I'm not sure why `? true : false` part is needed since it already returns `true`/`false` AFAIK.